tox.ini

```diff
@@ -12,6 +12,7 @@ statistics = True
 select = E901,E999,F821,F822,F823
 
 [testenv]
+passenv = *
```

Comment:
For my understanding, what does this do? Is this some wild card expression to allow all env variables to be passed ?

Reply:
Exactly. Adding this will pass in the env variables set from the github actions file into the testing environments for tox.

Reply:
As mentioned IMO, can we eliminate the usage tox in favor of Github Actions?
Well we don't have to do it in this PR, maybe in later stage as well.